src/assets/styles/layout.scss

```diff
@@ -9,7 +9,9 @@
   position: sticky;
   top: 0;
   width: 100%;
-  z-index: 2;
+  // place app header on top of dialogs
+  // and therefore still usable when a dialog is displayed
+  z-index: 1000;
```

Comment:
I moved this here because this app-header block already existed (no need to have both global and local css config).